tools/port_scanner.py

```
@@ -1,5 +1,7 @@
-import python_nmap as nmap  # type: ignore
-
+try:
+    import python_nmap as nmap  # type: ignore
+except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)